src/main/java/org/fundacionjala/coding/carlos/highestandlowest/CalculateNumbers.java

```diff
@@ -0,0 +1,28 @@
+package org.fundacionjala.coding.carlos.highestandlowest;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class CalculateNumbers {
+
+    public List<Integer> colectNumbers(String numbers) {
+        List<Integer> numbersList = new ArrayList<>();
+        String[] splitNumbers = numbers.split(" ");
+        for (int i=0; i<splitNumbers.length; i++){
+            numbersList.add(Integer.parseInt(splitNumbers[i]));
+        }
+        return numbersList;
+    }
+
+    public String highestAndLowest(String numbers) {
+       if (numbers.length()<=1){
+           return numbers;
+       }
+       List<Integer> numbersList = colectNumbers(numbers);
+       Collections.sort(numbersList);
```

Comment:
Sorting operations Sorting operations are too expensive to compute with large collections, so please consider using `Collections.max()` and `Collections.min()` utility methods.

Reply:
Ok, thanks